src/Serializer/ArticleVersionNormalizer.php

```diff
@@ -260,6 +280,12 @@ private function denormalizeSnippet(array $article) : PromiseInterface
                 })->toArray();
             }
 
+            if ($object->getRelatedArticles()->notEmpty()) {
+                $data['relatedArticles'] = $object->getRelatedArticles()->map(function (Article $relatedArticle) use ($format, $context) {
+                    return $this->normalizer->normalize($relatedArticle, $format, $context);
```

Comment:
`$context['snippet']` should be set to `true`, though easier to use the helper like https://github.com/elifesciences/api-sdk-php/blob/master/src/Serializer/CollectionNormalizer.php#L170.